site/_posts/2020-12-08-xkit-and-fusionauth.md

```diff
@@ -0,0 +1,107 @@
+
+<!--more-->
+
+We've written this post to lay out how you can use our services together to simplify your auth and build native integrations into your app faster. No more telling your customers that the integrations they've been asking for are "on the roadmap". Follow the steps below and you can ship them in no time.
+
+
+## What is Xkit
+
+Xkit is a SaaS platform which makes integrating third party systems a snap. Suppose you are writing a recipe management application and are going to sell it for big money to all the cooks of the world. After some market research, you realize that you want to integrate with other services. Your users are clamoring for the ability to export the steps of a recipe to a Trello board for sharing and Dropbox for backups. These are all services with APIs. 
+
+End users can give your application access to their accounts with these services, but that takes some coding. There's also a fair bit of hoop jumping: setting up API keys and OAuth consent screens, among other things.
+
+This is the problem which Xkit solves. If Xkit has a connection ([here's a list](https://docs.xkit.co/docs/connecting-with-apps-overview) of supported services), your application can, after Xkit is installed configured, allow a user to connect to a service. I (Dan) was able to connect to Trello to my app in about an hour. The user experience of connecting the external application is smooth and far better than something I could whip up in a day, let alone an hour.
+
+## Xkit and FusionAuth integration
+
+### Install FusionAuth
+
+FusionAuth offers [a number of different methods you can use to install the service](https://fusionauth.io/docs/v1/tech/installation-guide/). Once you've installed, there's a Setup Wizard to walk you through the next steps. You'll need to create your application in the FusionAuth interface and then add a few elements to your application code base to fully implement the FusionAuth login flow. A [full, detailed setup guide](https://fusionauth.io/docs/v1/tech/5-minute-setup-guide/) is also available. Feel free to create additional [users via the FusionAuth administrative user interface](https://fusionauth.io/docs/v1/tech/core-concepts/users/).
+
+Once this is installed you have a full featured user management system, ready to go. APIs to control everything, multi factor authentication, consent management, SAML, OIDC, and more, hosted wherever you want. There's also FusionAuth cloud, a managed services offering, if you don't want to host FusionAuth yourself.
```

Comment:
I'd reword this to "Once FusionAuth is installed and configured, you have a full featured user management system, ready to go."